impeller/geometry/geometry_unittests.cc

```diff
@@ -2090,5 +2092,36 @@ TEST(GeometryTest, Gradient) {
   }
 }
 
+TEST(GeometryTest, HalfConversions) {
+#ifndef FML_OS_WIN
```

Comment:
```suggestion
#ifdef FML_OS_WIN
  GTEST_SKIP() << "The reason this doesn't work on Windows";
```

And then remove the `#endif` below. This helps document why we're skipping it and makes it clear at execution time that the test is not actually doing anything on Windows.

Reply:
I can try this but I don't believe the half precision floating point literals below even compile on windows.